### PR TITLE
Candidates

### PR DESCRIPTION
**Pull request by vchepurnaya**
- add list of candidates
- add candidates card
- add filters to the list of candidates
- add status bar in card component
- add information about candidate
- add validation to the fields
- add candidates reviews 
- stylized card of candidate

### CODE REVIEW

File: src/components/candidates/candidate_card/candidate_card.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    },
+    {
+        value: 'Any time',
+        label: 'Any time',
+    }
+];
+
+const CandidateCard: React.FC = () => {
+    const classes = useStyles();
+    let {url} = useRouteMatch();
```
Comment (ZmicerBoksha) [inline src/components/candidates/candidate_card/candidate_card.tsx]:
```suggestion
    const { url } = useRouteMatch();
```

File: src/components/candidates/candidate_card/candidate_card.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    const [{ data: candidateInfo, loading: getLoading, error: getError }] = useAxios(
+        `${url}`
```
Comment (ZmicerBoksha) [inline src/components/candidates/candidate_card/candidate_card.tsx]:
```suggestion
        url
```

Comment (ZmicerBoksha) [reply src/components/candidates/candidate_card/candidate_card.tsx]: const [{ data: candidateInfo, loading: getLoading, error: getError }] = useAxios(url)

File: src/components/candidates/candidate_card/candidate_review/candidate_review.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                    }
+                    skill={'soft'}
+                />
+                {/**
+                 * here should be GET request to get reviews from BD
+                 **/}
+            </Paper>
+            <Paper className={classes.paper}>
+                <h2 className="card__review-title">Technical skills review</h2>
+                <CandidateSkills skill={'hard'}/>
```
Comment (ZmicerBoksha) [inline src/components/candidates/candidate_card/candidate_review/candidate_review.tsx]:
```suggestion
                <CandidateSkills skill='hard'/>
```

File: src/components/candidates/candidate_card/candidate_review/candidate_review.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+                            fullWidth
+                        >
+                            {englishLevels.map((option: TSelect) => (
+                                <MenuItem key={option.value} value={option.value}>
+                                    {option.label}
+                                </MenuItem>
+                            ))}
+                        </TextField>
+                    }
+                    skill={'soft'}
```
Comment (ZmicerBoksha) [inline src/components/candidates/candidate_card/candidate_review/candidate_review.tsx]:
```suggestion
                    skill='soft'
```

File: src/components/candidates/candidate_list/candidates_list.tsx [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    first_name: string,
+    last_name: string,
+    phone: string,
+    email: string,
+    skype: string,
+    english_level: string,
+    country: string,
+    city: string,
+    primary_skills: string,
+    other_technologies: Array<string>,
```
Comment (ZmicerBoksha) [inline src/components/candidates/candidate_list/candidates_list.tsx]: Better to use the same pattern for arrays in TS.
```suggestion
    other_technologies: string[],
```

File: src/components/candidates/candidate_list/candidates_list.tsx [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+    time: string
+};
+
+type TCandidateListProps = {
+    columns: Column<TCandidate>[]
+    defaultColumn: any
+}
+
+const CandidatesList: React.FC<TCandidateListProps> = ({columns, defaultColumn}) => {
+    let history = useHistory();
```
Comment (ZmicerBoksha) [inline src/components/candidates/candidate_list/candidates_list.tsx]:
```suggestion
    const history = useHistory();
```

**Comment by ZmicerBoksha** [comment]
If it is possible try to get rid of any type.

**Review by ZmicerBoksha — APPROVED** [review]

**Review by DmitryPateew — APPROVED** [review]

**Review by ArteGon — APPROVED** [review]

**Pull request merged**